code/modules/clothing/under/jobs/civilian.dm

```diff
@@ -1,13 +1,12 @@
 //Alphabetical order of civilian jobs.
 
 /obj/item/clothing/under/rank
-	rollable_sleeves = TRUE
 
 /obj/item/clothing/under/rank/bartender
 	desc = "It looks like it could use some more flair."
 	name = "bartender's uniform"
 	icon_state = "barman"
-	rollable_sleeves = FALSE
+	adjustment_variants = list()
```

Comment:
should this not be null

Reply:
i mean that you check if theres a lenth for the lst already so why not have this null so you dont create an empty list

Reply:
oh i suppose. I was just assuming that when you have a list, to make it null you make it empty